### ISSUE TITLE
Something went wrong with LiveReactload initialization

### ISSUE THREAD

**Issue by gilbarbara** [milankinen/livereactload#25]
Hey,

I saw a issue with this message (#18) but I think I have a different situation.
I'm using `react` with `react-router` and `react-bootstrap` and I initially tried you gulp example but I got some errors from the `react-bootstrap` refs.
So I tried using `livereactload` as a global transform and I get "Something went wrong with LiveReactload initialization" 9 times on the initial load. It works after these warnings but I'm curious why!

This is my setup:

```js
gulp.task('jswatch', function () {
	var bundler = browserify({
		entries: path.join(__dirname, '/app/scripts/main.js'),
		basedir: __dirname,
		insertGlobals: true,
		cache: {},
		debug: true,
		packageCache: {},
		fullPaths: true,
		transform: [['livereactload', { global: true }], 'reactify'],
		extensions: ['.jsx']
	});

	bundler = watchify(bundler);
	lrload.listen();

	var rebundle = function () {
		var stream = bundler.bundle();
		stream.on('error', $.util.log);

		stream
			.pipe(source('app.js'))
			.pipe(gulp.dest('.tmp/scripts'))
			.pipe(lrload.gulpnotify);
	};

	bundler.on('update', rebundle);
	return rebundle();
});
```

Thanks!

**Comment by milankinen** [comment]
I think I got it now. Accidentally tried to inject LiveReactload to transient dependencies (which were required during the initialization). Fix coming: https://github.com/milankinen/livereactload/commit/fcd6929b88981c578f6ef3d3b6cb370fc09247a5

**Comment by gilbarbara** [comment]
I changed that line and it worked!

Thanks

**Comment by milankinen** [comment]
Should be fixed in `0.4.0`. I'll open this if problem still exists
